### DISCUSSION TITLE
What times are used in minTemperature and maxTemperature in raw gridpoint data?

### DISCUSSION THREAD

**Discussion by mbg206** [weather-gov/api#676] [Q&A] [answered]
I notice that the `validTime` property in the `minTemperature` and `maxTemperature` values show a time period for about every 12hrs, but what exactly decides the start times for these periods? (Time zone, actual forecasted temperature, does DST affect, etc.)

**Comment by sullynole** [comment, marked as answer]
The raw gridpoint data values are grouped into periods with the same value.  Max and min temperature fields are a bit unique as they are usually forecast for 12h periods (max daytime hours, min nighttime hours) by WFO forecasters.
